docs/usage.md

```diff
@@ -8,6 +8,13 @@
   - [-profile](#-profile)
   - [--input](#--input)
   - [--split_fastq](#--split_fastq)
+  - [--trim_fastq](#--trim_fastq)
+  - [--clip_r1](#--clip_r1)
+  - [--clip_r2](#--clip_r2)
+  - [--three_prime_clip_r1](--three_prime_clip_r1)
+  - [--three_prime_clip_r2](--three_prime_clip_r2)
+  - [--trim_nextseq](--trim_nextseq)
+  - [--save_trimmed](--save_trimmed)
```

Comment:
You added that sections here in the TOC, but nothing in the document itself.
Some text could be good to explain these params.